docs/dotprompt.md

```diff
@@ -287,10 +287,13 @@ import {
   loadPromptUrl,
   defineDotprompt,
 } from '@genkit-ai/dotprompt';
+import path from 'path';
 import { z } from 'zod';
 
 // Load a prompt from a file
-const myPrompt = await loadPromptFile('./path/to/my_prompt.prompt');
+const myPrompt = await loadPromptFile(
+  path.resolve(__dirname, './path/to/my_prompt.prompt')
+);
```

Comment:
Not sure if this the preferred way or if this should be mentioned as an alternative.

Reply:
yeah, I'm a bit torn, but I think this way is more flexible, if a little more verbose. I think it's good.